charts/sftpgo/values.yaml

```diff
@@ -163,6 +163,57 @@ services: {}
   #       nodePort:
   #   externalTrafficPolicy:
 
+ingress:
```

Comment:
I'd prefer putting the component name first, eg. `api.ingress.enabled`.

Reply:
Hm … For me, the UI and API are part of sftpgo and not individual components. Anyway, in the interest of having a single chart, I can change this.